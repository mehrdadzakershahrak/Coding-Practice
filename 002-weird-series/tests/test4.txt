11 37 200
