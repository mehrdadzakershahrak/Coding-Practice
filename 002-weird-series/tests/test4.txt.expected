0
11
22
33
37
44
48
55
59
66
70
74
77
81
85
88
92
96
99
103
107
110
111
114
118
121
122
125
129
132
133
136
140
143
144
147
148
151
154
155
158
159
162
165
166
169
170
173
176
177
180
181
184
185
187
188
191
192
195
196
198
199
202
203
206
207
209
210
213
214
217
218
220
221
222
224
225
228
229
231
232
233
235
236
239
240
242
243
244
246
247
250
251
253
254
255
257
258
259
261
262
264
265
266
268
269
270
272
273
275
276
277
279
280
281
283
284
286
287
288
290
291
292
294
295
296
297
298
299
301
302
303
305
306
307
308
309
310
312
313
314
316
317
318
319
320
321
323
324
325
327
328
329
330
331
332
333
334
335
336
338
339
340
341
342
343
344
345
346
347
349
350
351
352
353
354
355
356
357
358
360
361
362
363
364
365
366
367
368
369
370
371
372
373
374
375
376
377
378
379
